webapp/src/pages/[resourceSlug]/onboarding/index.tsx

```diff
@@ -0,0 +1,83 @@
+import * as API from '@api';
+import { ChevronRightIcon } from '@heroicons/react/20/solid';
+import OnboardingSelect from 'components/onboarding/OnboardingSelect';
+import { useRouter } from 'next/router';
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+interface FormValues {
+	role: { label: string; value: string };
+}
+
+const Onboarding = () => {
+	const { control, handleSubmit } = useForm<FormValues>({
+		defaultValues: {
+			role: null
+		}
+	});
+	const router = useRouter();
+
+	const onSubmit = async (data: FormValues) => {
+		await API.updateRole(
+			{
+				role: data.role.value
+			},
+			null,
+			null,
+			router
+		);
+	};
+
+	return (
+		<main className='flex items-center flex-col justify-center flex-1'>
+			<img
+				src='/images/agentcloud-full-black-bg-trans.png'
+				alt='Agentcloud'
+				className='h-14 md:h-20'
+			/>
+			<section className='bg-white w-full max-w-[950px] rounded-lg shadow-sm p-8 flex flex-col gap-5'>
+				<h1 className='font-bold text-xl md:text-2xl'>Welcome to AgentCloud! 🎉</h1>
+				<div className='text-gray-600 text-lg sm:text-xl'>
+					Let&apos;s begin by getting to know your role.
+				</div>
+				<form
+					onSubmit={handleSubmit(onSubmit)}
+					className='flex flex-col gap-4 text-lg sm:text-2xl items-center'
+				>
+					<div className='flex flex-col sm:flex-row gap-2 w-full sm:items-center'>
+						<p className='w-fit font-extrabold'>I am a</p>
+						<div className='w-fit'>
+							<OnboardingSelect<FormValues>
+								options={[
+									{ value: null, label: null },
```

Comment:
This blank options looks a little strange, I suggest removal. 
![image](https://github.com/user-attachments/assets/320a526c-a4e9-4dfd-80e8-c24b0f7f9507)

:point_up: Thoughts? @anada10 and does Misri have github so I can tag her for these kind of questions?

Reply:
removed it  as its not in the figma design